### PR TITLE
General formating fixes for GDScript scripting pages

### PR DESCRIPTION
**Pull request by YeldhamDev**
Numerous consistency fixes, as well as styleguide compliance changes.

### CODE REVIEW

File: getting_started/scripting/gdscript/static_typing.rst [hunk 1]
```diff
@@ -387,17 +399,21 @@ Two scripts can’t depend on each other in a cyclic fashion:
 ::
 
     # Player.gd
+
     extends Area2D
     class_name Player
 
+
```
Comment (Calinou) [inline getting_started/scripting/gdscript/static_typing.rst]: The GDScript style guide only requires separating class and function declarations by two blank lines, not the first variable after `class_name`.

File: getting_started/scripting/gdscript/static_typing.rst [hunk 2]
```diff
@@ -387,17 +399,21 @@ Two scripts can’t depend on each other in a cyclic fashion:
...
     var rifle: Rifle
 
 ::
 
     # Rifle.gd
+
     extends Area2D
     class_name Rifle
 
+
```
Comment (Calinou) [inline getting_started/scripting/gdscript/static_typing.rst]: Same as above.

**Comment by YeldhamDev** [comment]
@Calinou If you see the styleguide page, it actually shows 2 blank lines for `extends`/`class_name`. Those lines are also in the engine's script templates.

**Comment by Calinou** [comment]
@YeldhamDev I see, it makes sense to merge this as-is then.

**Comment by Calinou** [comment]
Thanks!

**Pull request merged**